api/discovery/aggregated_v2beta1.json

```diff
@@ -1693,82 +1693,6 @@
             }
           ],
           "version": "v1"
-        },
```

Comment:
Are these changes intentional?

Reply:
Looks like something in this PR is interacting with schema gen in a way that's causing this removal. We need to find and fix.